python/semantic_kernel/connectors/ai/open_ai/prompt_execution_settings/open_ai_prompt_execution_settings.py

```diff
@@ -96,6 +96,19 @@ class OpenAIChatPromptExecutionSettings(OpenAIPromptExecutionSettings):
         dict[str, Any] | None,
         Field(description="Additional options to pass when streaming is used. Do not set this manually."),
     ] = None
+    max_completion_tokens: Annotated[
```

Comment:
My only other suggestion for this (along with fixing the unit tests), is that we should add a concept sample to show how to leverage the o1 model -- and make lots of comments around what is/ what is not supported:
- developer message instead of system
- parallel tool calls not yet supported (should set PES `parallel_tool_calls = False`
- unsupported API parameters: temperature, top_p, presence_penalty, frequency_penalty, logprobs, top_logprobs, logit_bias

Reply:
Understood. I will add a note in the comments indicating that this is a property specific to o1. Additionally, I will include examples of how to use o1 in directories like /python/samples/concepts/reasoning/**.

Reply:
I have added samples!